addons/containerd/1.5.11/install.sh

```diff
@@ -78,8 +78,8 @@ function containerd_install() {
         # If using the internal load balancer the Kubernetes API server will be unavailable until
         # kubelet starts the HAProxy static pod. This check ensures the Kubernetes API server
         # is available before proceeeding.
-        # ".." is needed becasue addons can have a CRD names "nodes", like nodes.longhorn.io
-        try_5m kubectl --kubeconfig=/etc/kubernetes/kubelet.conf get nodes..
+        # "nodes.v1." is needed becasue addons can have a CRD names "nodes", like nodes.longhorn.io
+        try_5m kubectl --kubeconfig=/etc/kubernetes/kubelet.conf get nodes.v1.
```

Comment:
why has this not been changed in the addon template?

Reply:
And why 1.5.11 but not 1.6, or the other 1.5, and so on?

Reply:
yeah made a different PR to fix all the older versions and template for the upcoming versions https://github.com/replicatedhq/kURL/pull/3310/files#diff-8a7133184a76cd0383e373a9cff3a1b6c7c5f68233b702bc3ae0e101d2f003ffL81-L82

Reply:
handled in https://github.com/replicatedhq/kURL/pull/3310